### PR TITLE
few Navigation and PeopleList styling changes

### PR DESCRIPTION
**Pull request by Adrode**
Changed PeopleList container to "section", added top margin to section to match Figma design.

Added gap in Navigation, between LogoButtonContainer and Input to imporve responsiveness

### CODE REVIEW

File: src/components/Navigation/styled.js [hunk 1]
```diff
@@ -27,6 +28,10 @@ export const Wrapper = styled.div`
     justify-content: center;
     gap: 24px;
   }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.phone}px) {
```
Comment (Urbork) [inline src/components/Navigation/styled.js]: Could you change it to new breakpoint from theme please?

**Review by Urbork — COMMENTED** [review]
It looks great. If you have time just update breakpoints to medium/small

**Review by mariuszmmm — COMMENTED** [review]
Please resolve conflicts.

**Review by mariuszmmm — APPROVED** [review]
It looks very good, even on small screens. Great job.

**Comment by Urbork** [comment]
Awesome, very neat and precise

**Pull request merged**